src/components/common/markets/dashboard-group.tsx

```
@@ -80,90 +79,99 @@ export const DashboardGroup = ({ markets, type, categoryName }: DashboardGroupPr
+          }
+        }
+
+        return (
+          <VStack spacing={4} w='full'>
+            {allRows}
           </VStack>
         )
 
       case DashboardGroupType.Compact:
-        const compactMarkets = markets.slice(0, 5)
         const gapSizeCompact = 12
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)